Category: Error, Message: System.InvalidOperationException: No connection string named 'EntitiesGeneral' could be found in the application config file.
   en System.Data.Entity.Internal.LazyInternalConnection.get_ConnectionHasModel()
   en System.Data.Entity.Internal.LazyInternalContext.InitializeContext()
   en System.Data.Entity.Internal.InternalContext.GetEntitySetAndBaseTypeForType(Type entityType)
   en System.Data.Entity.Internal.Linq.InternalSet`1.Initialize()
   en System.Data.Entity.Internal.Linq.InternalSet`1.get_InternalContext()
   en System.Data.Entity.Infrastructure.DbQuery`1.System.Linq.IQueryable.get_Provider()
   en System.Linq.Queryable.Select[TSource,TResult](IQueryable`1 source, Expression`1 selector)
   en Core.Erp.Data.General.tb_sis_Mensajes_sys_Data.Get_List_Mensajes_sys() en c:\Users\OBANDO\Desktop\PROYECTOS GITHUB\RPDDESKTOP\ERP\Core.Erp.Data\General\tb_sis_Mensajes_sys_Data.cs:línea 49
Category: Error, Message: System.InvalidOperationException: No connection string named 'EntitiesGeneral' could be found in the application config file.
   en System.Data.Entity.Internal.LazyInternalConnection.get_ConnectionHasModel()
   en System.Data.Entity.Internal.LazyInternalContext.InitializeContext()
   en System.Data.Entity.Internal.InternalContext.GetEntitySetAndBaseTypeForType(Type entityType)
   en System.Data.Entity.Internal.Linq.InternalSet`1.Initialize()
   en System.Data.Entity.Internal.Linq.InternalSet`1.get_InternalContext()
   en System.Data.Entity.Infrastructure.DbQuery`1.System.Linq.IQueryable.get_Provider()
   en System.Linq.Queryable.Where[TSource](IQueryable`1 source, Expression`1 predicate)
   en Core.Erp.Data.General.tb_Sucursal_Data.Get_List_Sucursal(Int32 IdEmpresa) en c:\Users\OBANDO\Desktop\PROYECTOS GITHUB\RPDDESKTOP\ERP\Core.Erp.Data\General\tb_Sucursal_Data.cs:línea 22
Category: Error, Message: Core.Erp.Info.Log_Exception.DalException ---> System.Exception: System.InvalidOperationException: No connection string named 'EntitiesGeneral' could be found in the application config file.
   en System.Data.Entity.Internal.LazyInternalConnection.get_ConnectionHasModel()
   en System.Data.Entity.Internal.LazyInternalContext.InitializeContext()
   en System.Data.Entity.Internal.InternalContext.GetEntitySetAndBaseTypeForType(Type entityType)
   en System.Data.Entity.Internal.Linq.InternalSet`1.Initialize()
   en System.Data.Entity.Internal.Linq.InternalSet`1.get_InternalContext()
   en System.Data.Entity.Infrastructure.DbQuery`1.System.Linq.IQueryable.get_Provider()
   en System.Linq.Queryable.Where[TSource](IQueryable`1 source, Expression`1 predicate)
   en Core.Erp.Data.General.tb_Sucursal_Data.Get_List_Sucursal(Int32 IdEmpresa) en c:\Users\OBANDO\Desktop\PROYECTOS GITHUB\RPDDESKTOP\ERP\Core.Erp.Data\General\tb_Sucursal_Data.cs:línea 22
   en Core.Erp.Data.General.tb_Sucursal_Data.Get_List_Sucursal(Int32 IdEmpresa) en c:\Users\OBANDO\Desktop\PROYECTOS GITHUB\RPDDESKTOP\ERP\Core.Erp.Data\General\tb_Sucursal_Data.cs:línea 59
   en Core.Erp.Business.General.tb_Sucursal_Bus.Get_List_Sucursal_Todos(Int32 IdEmpresa) en c:\Users\OBANDO\Desktop\PROYECTOS GITHUB\RPDDESKTOP\ERP\Core.Erp.Bussines\General\tb_Sucursal_Bus.cs:línea 68
   --- Fin del seguimiento de la pila de la excepción interna ---
   en Core.Erp.Business.General.tb_Sucursal_Bus.Get_List_Sucursal_Todos(Int32 IdEmpresa) en c:\Users\OBANDO\Desktop\PROYECTOS GITHUB\RPDDESKTOP\ERP\Core.Erp.Bussines\General\tb_Sucursal_Bus.cs:línea 75
   en Core.Erp.Reportes.Controles.UCCXC_MenuReportes.UCCXC_MenuReportes_Load(Object sender, EventArgs e) en c:\Users\OBANDO\Desktop\PROYECTOS GITHUB\RPDDESKTOP\ERP\Core.Erp.Reportes\Controles\UCCXC_MenuReportes.cs:línea 280
